spark/src/main/scala/org/apache/spark/sql/delta/DeltaSharedExceptions.scala

```diff
@@ -20,28 +20,67 @@ import scala.collection.JavaConverters._
 
 import org.antlr.v4.runtime.ParserRuleContext
 
+import org.apache.spark.QueryContext
 import org.apache.spark.sql.AnalysisException
 import org.apache.spark.sql.catalyst.parser.{ParseException, ParserUtils}
 import org.apache.spark.sql.catalyst.trees.Origin
 
 class DeltaAnalysisException(
```

Comment:
This change means that sometimes `DeltaAnalysisException` may not have an errorClass which was previously guaranteed. Other changes are attributed to this change in assumption.